### PR TITLE
Update enable-cloud-protection-microsoft-defender-antivirus.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the navigation as the options have changed

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 please copy review

**Comment by msbemba** [comment]
Ticket#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/10180

File: microsoft-365/security/defender-endpoint/enable-cloud-protection-microsoft-defender-antivirus.md [hunk 1]
```diff
@@ -51,15 +51,15 @@ For more information about the specific network-connectivity requirements to ens
 
 1. Go to the Microsoft Endpoint Manager admin center ([https://endpoint.microsoft.com](https://endpoint.microsoft.com)) and sign in.
 
-2. On the **Home** pane, select **Device configuration > Profiles**.
+2. On the **Home** pane, select **Devices**> **Configuration Profiles**.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/enable-cloud-protection-microsoft-defender-antivirus.md]:
```suggestion
2. On the **Home** pane, select **Devices** > **Configuration Profiles**.
```

File: microsoft-365/security/defender-endpoint/enable-cloud-protection-microsoft-defender-antivirus.md [hunk 2]
```diff
@@ -51,15 +51,15 @@ For more information about the specific network-connectivity requirements to ens
...
 
 3. Select the **Device restrictions** profile type you want to configure. If you need to create a new **Device restrictions** profile type, see [Configure device restriction settings in Microsoft Intune](/intune/device-restrictions-configure).
 
-4. Select **Properties** \> **Configuration settings: Edit** \> **Microsoft Defender Antivirus**.
+4. In the **Properties** section Click on **Edit** next to **Configuration settings** and click on **Microsoft Defender Antivirus**.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/enable-cloud-protection-microsoft-defender-antivirus.md]:
```suggestion
4. In the **Properties** section, select **Edit** next to **Configuration settings** and then select **Microsoft Defender Antivirus**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@denisebmsft Please merge this PR. Thanks

**Review by denisebmsft — APPROVED** [review]
Thank you @msbemba @JohanFreelancer9

**Pull request merged**